lnrpc/peersrpc/peers_server.go

```diff
@@ -254,6 +254,12 @@ func (s *Server) updateFeatures(currentfeatures *lnwire.RawFeatureVector,
 	for _, update := range updates {
 		bit := lnwire.FeatureBit(update.FeatureBit)
 
+		if name, known := lnwire.KnownFeature(bit); known {
```

Comment:
Would it be good to also disallow the removal of custom feature bits that were set through config? It feels a bit strange to hard-configure features that are then dynamically cleared via the rpc.

Reply:
Yeah agreed. Will add on next push!

Reply:
So the reason we can't modify feature bits known to LND is that wonky things could happen if the advertised bits don't match what LND implements.

But I don't think there's a similar concern for overriding custom feature bits set in the config.  And there may be use cases where we wish to temporarily override a feature and avoid a restart (e.g., disable feature X for the next channel open only).

Reply:
@morehouse that's a good point. However to keep it simple I would keep it as it is right now (disabling the option to change feature bits set in the config file).  

If there is any action that wants to override the default behaivour of the node (feature bits for a new channel for example) it should be exposed in the  relevant API call instead of having to play with the global state of the node.

Reply:
> And there may be use cases where we wish to temporarily override a feature and avoid a restart 

I wouldn't recommend setting in cofig at all for this use case? Just set/unset on the already available API.

Reply:
>Now that I've done this, I'm not sure that it's worth the lift because we need to thread our configured feature bits through to peersrpc, but it does seem more logically consistent.

I think it was worth it.